pkg/command/diagnose/service.go

```diff
@@ -75,7 +75,7 @@ knative-diagnose service <ksvc-name> -ns <namespace>\n`)
 func dumpToTables(f *Fetcher, verbose string) error {
 
 	if f.KSVC == nil {
-		return fmt.Errorf("Can't fetch any KSVC CR for the target service")
+		return fmt.Errorf("Can't fetch any KSVC CR for the target knative service")
```

Comment:
I wonder if this GIANT `dumpToTables` function could be split into multiple smaller helper functions focused on one aspect of the table?

Reply:
yes, Max, it should be.  
This PR is only to make the current `demo` code base works .

I am thinking of refactor it completely to setup a good basis for more people to join.